ini/teensy.ini

```
@@ -9,61 +9,71 @@
 #                               #
 #################################
 
+#
+# Teensy AVR
+#
+[teensy_avr]
+platform   = teensy
+extends    = common_avr8
+lib_ignore = ${env:common_avr8.lib_ignore}, NativeEthernet
```

You don't get the new compiler on the `teensy` platform without overriding `toolchain-atmelavr`.

```suggestion
[teensy_avr]
platform          = teensy
platform_packages = toolchain-atmelavr@~1.70300.0
extends           = common_avr8
lib_ignore        = ${env:common_avr8.lib_ignore}, NativeEthernet
```

---

It's not intentionally removed, it just keeps getting lost in my attempts to edit this file.